CONTRIBUTING.md

```diff
@@ -0,0 +1,114 @@
+<!-- 
+
+- Taking inspiration from https://github.com/zowe/docs-site/blob/master/README.md
+- Commenting out sections from that until we mature and need all their process
+
+-->
+<!-- uncomment once we get to the point of welcoming feedback 
+## Providing feedback
+
+Your feedback is essential in shaping the Zowe content experience. There are several ways to provide feedback:
+
+- If you see something incorrect or confusing in the docs, or have an enhancement idea to make the docs better, you can edit a page by clicking **Edit this page** at the top of the page to [open a Pull Request](https://docs.zowe.org/stable/contribute/contributing#sending-a-github-pull-request). You can also [open a GitHub issue](https://docs.zowe.org/stable/contribute/contributing#opening-an-issue-for-the-documentation) for the documentation team by clicking **Open doc issue**.
```

Comment:
Don't we need to change the links to our links here?

Reply:
And more below.

Reply:
no, these are also commented out for now